src/CommitmentsV2/SFA.DAS.CommitmentsV2.Api.Types/Responses/GetApprenticeshipUpdateResponse.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using SFA.DAS.CommitmentsV2.Types;
+
+namespace SFA.DAS.CommitmentsV2.Api.Types.Responses
+{
+    public class GetApprenticeshipUpdateResponse
+    {
+        public ApprenticeshipUpdate PendingApprenticeshipUpdate { get; set; }
+
+        public class ApprenticeshipUpdate
+        {
+            public long Id { get; set; }
+            public long ApprenticeshipId { get; set; }
+            public Originator Originator { get; set; }
```

Comment:
Can we convert the "Originator" field to type "Party", and convert it from Originator using an extension method like we have with other fields, etc. "Originator" is one of those types I'd like to get rid of in the long term, so we will keep it in the api for now but not expose it. I think using Party here is fine (it would never be TransferSender, of course, but other than that is ok).

Reply:
done